src/consts.ts

```diff
@@ -47,6 +46,7 @@ export let HubTypeNames = Object.keys(HubType).reduce((result: {[hubType: string
  * @property {number} CONTROL_PLUS_LARGE_MOTOR 46
  * @property {number} CONTROL_PLUS_XLARGE_MOTOR 47
  * @property {number} POWERED_UP_REMOTE_BUTTON 55
+ * @property {number} RSSI 56
  * @property {number} CONTROL_PLUS_ACCELEROMETER 58
  * @property {number} CONTROL_PLUS_TILT 59
```

Comment:
I did not include here the mystery device type 66 which is an interval device in the Boost Hub at port 0x46. Do you have any idea what that is?

Reply:
I don't - but I've sent off a couple of emails to ask. I think for the moment lets leave it off.

Reply:
I tested this on my own and this port seems to return 0 as a value in all 3 modes, so probably not interesting anyway.